### ISSUE TITLE
Installation command on readme

### ISSUE THREAD

**Issue by Bollos00** [EnderIce2/WhatsApp-Wine#1]
Hello there and thanks for the effort put in this project.

I was trying to run your script, but it seems that I cannot use `git clone` to clone a single file like this: `git clone https://github.com/EnderIce2/WhatsApp-Wine/whatsapp-wine.sh` (as it is in the readme), what I could do instead was to clone the role repo: `git clone https://github.com/EnderIce2/WhatsApp-Wine /tmp/WhatsApp-Wine` (clone into `/tmp/WhatsApp-Wine` directory) or wget the raw file: `wget https://raw.githubusercontent.com/EnderIce2/WhatsApp-Wine/master/whatsapp-wine.sh`.

I changed a little the command to paste on the terminal to this:
```
(git clone https://github.com/EnderIce2/WhatsApp-Wine /tmp/WhatsApp-Wine  && chmod +x /tmp/WhatsApp-Wine/whatsapp-wine.sh && /tmp/WhatsApp-Wine/whatsapp-wine.sh) ; rm -rf /tmp/WhatsApp-Wine
```

That way, it will clone the repository on the to `/tmp/Whats-Wine`, execute the script and, after all, remove the directory of the repository (even if the script does not run well). That way, this command will always work.

Let me know what you think, if you have something else on your mind.

**Comment by Bollos00** [comment]
By the way, I am having some issues to run it on manjaro (arch linux based). I think it installed well, but is unable to start. Here is the output I get:
```
╰─ (git clone https://github.com/EnderIce2/WhatsApp-Wine /tmp/WhatsApp-Wine  && chmod +x /tmp/WhatsApp-Wine/whatsapp-wine.sh && /tmp/WhatsApp-Wine/whatsapp-wine.sh) ; rm -rf /tmp/WhatsApp-Wine
Cloning into '/tmp/WhatsApp-Wine'...
remote: Enumerating objects: 50, done.
remote: Counting objects: 100% (50/50), done.
remote: Compressing objects: 100% (48/48), done.
remote: Total 50 (delta 22), reused 3 (delta 1), pack-reused 0
Receiving objects: 100% (50/50), 15.43 KiB | 3.86 MiB/s, done.
Resolving deltas: 100% (22/22), done.
SCRIPT VERSION 1.4.2
/home/bollos00/.config/WhatsApp-wine/installed
WhatsApp Wine is already installed (.whatsapp) [Script v1.4.2]
====================================
| Start WhatsApp          [S]      |
| Kill WhatsApp           [K]      |
| Winecfg                 [W]      |
| Winetricks              [T]      |
| Repair                  [R]      |
| Uninstall               [U]      |
| About This Script       [A]      |
====================================
> s
MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0


wine: Unhandled page fault on read access to 0000001100000135 at address 000000007BC28690 (thread 0248), starting debugger...


```

![Screenshot_20211125_163547](https://user-images.githubusercontent.com/53309405/143493473-df5b16d3-e1a0-4c3c-8117-789eff6eef54.png)

It creates this black windows with this dialog...

**Comment by EnderIce2** [comment]
I fixed the typo, and I see that wine-staging 6.22 is not working with WhatsApp (page fault and missing ntdll.dll functions), I will have to fix this problem by making this script install an older version of wine. For now, I don't have enough time to do this.
